packages/webdoc-default-template/tmpl/components/member.tmpl

```diff
@@ -4,7 +4,7 @@
  */
 const doc = obj;
 ?>
-<div class="member">
+<div class="member" id="<?js= doc.name ?>">
```

Comment:
Is this **always** globally unique?

Reply:
Yes, this is unique within the generated page.